pkg/animal/birds.go

```diff
@@ -5,17 +5,11 @@ import (
 	"github.com/ironarachne/world/pkg/size"
 )
 
-func getBirds() []Animal {
-	birds := []Animal{
+func getBirds() []Species {
+	birds := []Species{
```

Comment:
In some of the other files you use a different method to initialize these. I haven't looked to closely but would you like them to be refactored here or in another PR if they can use be initialized using the loop? I'll comment on the other method I'm talking about.